### ISSUE TITLE
Support Question: How does this compare to the Drupal Bootstrap Module?

### ISSUE THREAD

**Issue by shawnholt** [drupalprojects/kalatheme#185]
I see a lot of traction with the drupal bootstrap module: https://www.drupal.org/project/bootstrap , but there have been no recent stable releases and many issues.  How is Kalatheme different?  Why are issues/statistics/etc. not on drupal.org like most modules?  Are the statistics accurate?

IMO I think it would get a lot more consideration if it were more active there.

**Comment by labboy0276** [comment]
Hi Shawn,

I have used the bootstrap theme in the past.  From what I remember about it, it is just a typical bootstrap plug and play theme.  Which is fine if that is what you need.  Kalatheme is more robust then just a plug and play theme.  The laundry list is long, but I suggest you read this:

https://github.com/drupalprojects/kalatheme/wiki

The reason the module doesn't have issues on DO anymore, is that GitHub promotes more of a universal collaboration space.  DO has always been quasi limiting as to who can do what.  If you have maintained your own module, then you would know how much different it is just to roll a release then normal GitHub based workflows.  Also having it on GitHub promotes collaboration outside of Drupal itself.  Which is important if you want to stay up with newer technologies in general.  There has been a small push to get modules into GitHub (http://hubdrop.org/ && https://github.com/drupalprojects), even the bootstrap theme is in there.  The stats are accurate though, the modules themselves are still released through DO.  

If you have any other questions, please don't hesitate to ask.

**Comment by shawnholt** [comment]
Thanks @labboy0276 for the prompt and thoughtful reply.  I do hope that drupal migrates to/syncs with github.

**I'm still a bit puzzled if stats are correct.**  There are very few bootstrap - drupal base theme options.  Given how much more robust, stable and actively developed kalatheme is, **I'm really curious why stats show 1K vs. 65k installs!** Is this just a marketing problem or are there other potential issues.  I know bootstrap is accused of not being "lean" so I can't imagine that is the issue.  

I'm hoping more insights could help other system-builders like myself make smarter choices in a base theme as it is a pretty important decision.

Note I build a site with bootatrap and feeling stuck in 3.0 and poor navbar support.

**Comment by labboy0276** [comment]
Hello Again

The installs vs downloads dilemma has always been an issue.  The stats themselves can be questionable as a lot of sites do not use the update module.  I would look at downloads vs installs.  Also, bootstrap theme itself has been around for over 5 years longer then Kalatheme.  If I remember right, they were one of the first Bootstrap themes and so, they had a good running start.  Also to name your theme bootstrap is a more easily find-able marketable asset then something names Kalatheme.  

My experience is this, if you are running a modern stack with Panopoly (or just Panels) then run Kalatheme.  If you are running an older setup with just blocks and context, then use the Bootstrap theme.

**Comment by pirog** [comment]
Kalatheme was built specifically for panels/panopoly whereas Bootstrap is more generic. Kalatheme also splits the Bootstrap-for-panels market with Radix. 

The Bootstrap project is not panels specific and of course having the Bootstrap namespace helps a lot. 

Another thing to note re: Bootstrap the framework and the ongoing development of a lot of the drupal bootstrap themes is that because Drupal contrib is generally only compatible with embarrassingly low versions of JQuery a lot of work has to go into getting contrib up to date before pulling in new bootstrap features. But @robloach may know more on this specific point since he has been a hero on this initiative.

**Comment by RobLoach** [comment]
> @pirog  a lot of work has to go into getting contrib up to date

Yup! The latest [Bootstrap theme](https://www.drupal.org/project/bootstrap) recommends using jQuery 1.9+ through jQuery Update, and is compatibile with the latest version of Bootstrap (either through CDN, or [Bootstrap Library](https://drupal.org/project/bootstrap_library)).

> @pirog Bootstrap is more generic.

Correct. All third-party support related to the Bootstrap theme are off-loaded to supporting modules, like [Panels Bootstrap Layouts](https://www.drupal.org/project/panels_bootstrap_layouts), or [Display Suite Bootstrap Layouts](https://www.drupal.org/project/ds_bootstrap_layouts). Kalatheme has that support built-in.

Have a look at the [Bootstrap related modules list](https://www.drupal.org/node/2011034) for some supporting modules that help out not only if you're on the Bootstrap theme, but also if you're on Kalatheme.

> @shawnholt I see a lot of traction with the drupal bootstrap module: https://www.drupal.org/project/bootstrap , but there have been no recent stable releases and many issues.

While it's not technically a "stable" release, you can still use the [7.x-3.1-beta2](https://www.drupal.org/node/2312697), or even [7.x-3.x-dev](https://www.drupal.org/node/1840986). A lot of work has gone on for better integration, stability, new features and support for the latest version of Bootstrap. I highly recommend you check it out. While it isn't technically stable, it still might meet your needs.

**Comment by andrewmallis** [comment]
let's roll some of this into the wiki, perhaps in https://github.com/drupalprojects/kalatheme/wiki/Tips,-Tricks,-and-FAQ

**Comment by astuhr** [comment]
Do we still need to install the Panapoly Theme module (and its radix layouts module dependency)? All of the Panapoly panels layouts are listed as deprecated (I assume to the radix ones) and the Radix Layouts module page indicates that radix is a part of Panapoly now, although I think they are talking about the distro. I am getting some errors when using the responsive Panels layouts and wonder if it is because I am confusing my system. Any help is greatly appreciated. Thanks

**Comment by labboy0276** [comment]
Hello,

You don't need Radix or Panopoly to use Kalatheme.

On a vanilla install you will need these modules to get this to work:
JQuery Update
Panels
Libraries
Ctools
Views (Optional)

I have a site that has just those installed with Kalatheme and it works gloriously.
